test/test-trace-api.js

```diff
@@ -94,13 +101,14 @@ describe('Trace Interface', function() {
     
     before(function() {
       traceAPI.enable_(agent);
-      common.init(traceAPI);
-      common.avoidTraceWriterAuth(traceAPI);
+      traceAPI.private_().traceWriter.request = request;
```

Comment:
`private_` is meant to disappear in a forthcoming PR, since the several layers of indirection are being merged.

In the particular line being highlighted, however, the `private_` access isn't needed since the TraceWriter can be accessed as a singleton.